k8s/tf/200_single_state/eu-central-1/eks.tf

```diff
@@ -1,71 +1,47 @@
 locals {
   cluster_features = {
-    "prometheus"         = false
-    "flux"               = true
-    "flux_helm_operator" = true
-    "external_secrets"   = false
+    "aws_calico"         = true
+    "configmapsecrets"   = true
+    "external_secrets"   = true
+    "prometheus"         = true
+    "fluentd_papertrail" = true
   }
 
-  flux_settings = {
-    "git.url"    = "git@github.com:mozilla-it/sumo-infra"
-    "git.path"   = "k8s/flux/eu-central-1"
-    "git.branch" = "master"
-  }
 
-  worker_groups = [
-    {
-      name                  = "k8s-worker-green"
```

Comment:
We use these during upgrades, curious if you wanted to keep one at 0 or just add it when we need to upgrade?

Reply:
Indeed it can be done in both ways, I keep with just one and add a new one when upgrading to keep the consistency with other configs as [itse-apps](https://github.com/mozilla-it/itse-apps-prod-1-infra/blob/main/terraform/shared/main.tf#L24).